content/docs/hardware/general/models/_index.de.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Modellauswahl
+weight: 10
+---
+
+Wenn du einen OpenBikeSensor Abstandsmesser bauen möchtest, musst du zunächst
```

Comment:
`OpenBikeSensor Abstandsmesser` - I assume this should be only `OpenBikeSensor`? The term `Abstandsmesser` only appears here

Reply:
Ja, das ist die Problematik mit dem Begriff. Ich finds nicht so schlimm, wenn wir auch das Gerät so nennen, aber @madiko ist das immer wichtig. Vielleicht hast du ja einen Vorschlag wie wir das textuell differenzieren können, @madiko ?

Reply:
Let's not discuss this in this PR, but in #82 which I've just created. We'll clean it up later, ok?